tests/migration/online_migration/pre_migration.pm

```diff
@@ -19,15 +19,19 @@ use x11utils 'turn_off_gnome_show_banner';
 
 sub check_or_install_packages {
     if (get_var("FULL_UPDATE") || get_var("MINIMAL_UPDATE")) {
-        # if system is fully updated or even minimal patch applied, all necessary packages for online migration should be installed
-        # check if the packages was installed along with update
-        my $output = script_output "rpm -qa yast2-migration zypper-migration-plugin rollback-helper | sort";
-        if ($output !~ /rollback-helper.*?yast2-migration.*?zypper-migration-plugin/s) {
-            record_soft_failure 'bsc#982150: migration packages were not installed along with system update. Installing missed package to continue the test';
-            zypper_call "in yast2-migration zypper-migration-plugin rollback-helper snapper";
+        if (is_leap_migration) {
+            # https://bugzilla.suse.com/show_bug.cgi?id=1197268#c2
+            record_soft_failure('bsc#1197268', 'suseconnect-ng obsoletes zypper-migration-plugin in leap to sle migration');
```

Comment:
This should not softfail unconditionally. Please detect whether this bug applies and only then record a softfail and work around it.

Reply:
OK. we'd remove it once bsc#1197268 was verified as not a bug.